CryptKeeperZK/crypt-keeper-extension
# fix: create random identity

## Explanation

Currently, there is an issue when creating new `random` identity. It has the same text as `interrep` with twitter provider and nonce 0.

As a solution, I changed logic for message signature depending on params.
Other changes are described below:

- [x] Add tests for create identity modal
- [x] Use specific message for signature
- [x] Run prettier on related files
- [x] Minor rename refactoring

## More Information

Fixes #81

## Screenshots/Screencaps



### Before

![image](https://user-images.githubusercontent.com/14254374/217670426-d133337e-82d4-4023-b303-0050b87563d8.png)
![image](https://user-images.githubusercontent.com/14254374/217670451-5078e7fa-ac32-4822-8cbe-ccf85fa7c470.png)

### After

![image](https://user-images.githubusercontent.com/14254374/217670336-919a4819-ca40-4626-8bed-cbe95a7a122e.png)
![image](https://user-images.githubusercontent.com/14254374/217670393-b1958624-931b-4b40-b439-d875355914e3.png)

## Manual Testing Steps

1. Open extension
2. Try to create `random` or `interrep` identity
3. Check the messages you sign with metamask
4. Check the identity list labels

## Pre-Merge Checklist


### + If there are functional changes:


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
